src/models/stackedAreaChart.js

```diff
@@ -375,7 +374,8 @@ nv.models.stackedAreaChart = function() {
                 //If we are in 'expand' mode, force the format to be a percentage.
                 var valueFormatter = (stacked.style() == 'expand') ?
                     function(d,i) {return d3.format(".1%")(d);} :
-                    function(d,i) {return yAxis.tickFormat()(d); };
+                    interactiveLayer.tooltip.valueFormatter(); 
```

Comment:
Same idea as previous iteration, but using `interactiveLayer.tooltip` instead.

Users will be able to use `headerFormatter()` and `valueFormatter()` like in the following example:

        chart.interactiveLayer.tooltip
            .headerFormatter(function (d) {
                return 'Header: ' + d;
            })
            .valueFormatter(function (d, i) {
                return d + ' monkeys';
            });

If not specified, the formatters will default to the axis `tickFormat()` functions (specified at line 43) as before.